src/main/java/graphql/execution/instrumentation/parameters/InstrumentationFieldParameters.java

```diff
@@ -27,7 +24,4 @@ public GraphQLFieldDefinition getField() {
         return fieldDef;
     }
 
-    public DataFetchingEnvironment getEnvironment() {
-        return environment;
-    }
 }
```

Comment:
In a split world, the data fetching environment is not available any more at this stage